client/components/Querries.tsx

```diff
@@ -0,0 +1,129 @@
+                <div
+                    aria-hidden="true"
+                    className={`loggerBtn${time ? ' active' : ''}`}
+                    onClick={() => {
+                        sortByTime();
+                        setToggle('time');
+                    }}
+                >
+                    Time {arrowTime}
+                </div>
+                <div
+                    aria-hidden="true"
+                    className={`loggerBtn${depth ? ' active' : ''}`}
+                    onClick={() => {
+                        sortByDepth();
+                        setToggle('depth');
+                    }}
+                >
+                    Depth {arrowDepth}
+                </div>
+                <div
+                    aria-hidden="true"
+                    className={`loggerBtn${complexity ? ' active' : ''}`}
+                    onClick={() => {
+                        sortByComplexity();
+                        setToggle('complexity');
+                    }}
+                >
+                    Complexity {arrowComplexity}
+                </div>
+            </div>
+            {/* -------this is left for the future project implementation. Waiting for instruction
+                {projects?.map((project: Project) => (
+                <div aria-hidden="true" onClick={() => test(project)}>
+                    <Collapsible
+                        trigger={<span style={{ padding: '10px 50px' }}>{project.name}</span>}
+                        className="projectCard"
+                    >
+                        <div>
+                            {project.queries.map((query: ProjectQuery) => (
+                                <div className="queryProps">
+                                    <div>
+                                        <div className="label">Name: </div>
+                                        <div className="value">{query.name}</div>
+                                    </div>
+                                    <div>
+                                        <div className="label">Time: </div>
+                                        <div className="value">{query.time}</div>
+                                    </div>
+                                    <div>
+                                        <div className="label">Depth: </div>
+                                        <div className="value">{query.depth}</div>
+                                    </div>
+                                    <div>
+                                        <div className="label">Complexity: </div>
+                                        <div className="value">{query.complexity}</div>
+                                    </div>
+                                </div>
+                            ))}
+                        </div>
+                    </Collapsible>
+                </div>
+            ))} */}
+            {queries?.map((query: ProjectQuery) => (
+                <div>
```

Comment:
Can the scroll bars be removed from the Query panel? I think it would look cleaner without these or with a more modern style.

Reply:
Will look into that!